vcd/resource_vcd_independent_disk.go

```diff
@@ -77,9 +123,26 @@ var busSubTypes = map[string]string{
 	"ahci":        "vmware.sata.ahci",
 }
 
+var busSubTypesFromValues = map[string]string{
+	"IDE":              "ide",
+	"buslogic":         "buslogic",
+	"lsilogic":         "lsilogic",
+	"lsilogicsas":      "lsilogicsas",
+	"VirtualSCSI":      "virtualscsi",
+	"vmware.sata.ahci": "ahci",
+}
+
 func resourceVcdIndependentDiskCreate(d *schema.ResourceData, meta interface{}) error {
 	vcdClient := meta.(*VCDClient)
 
+	size, sizeProvided := d.GetOk("size")
+	sizeInBytes, sizeInBytesProvided := d.GetOk("size_in_bytes")
```

Comment:
Looks like this should be commented out as well together with the below block.

Reply:
I didn't wanted cause mess up code and this code won't work.

Reply:
What is the use of leaving a half-baked size_in_bytes approach? We agreed to remove it for now and return to it in 2.6.0.

Reply:
this already commented out